astropy/coordinates/angle_formats.py

```diff
@@ -412,11 +413,15 @@ def degrees_to_dms(d):
 def dms_to_degrees(d, m, s=None):
     """
     Convert degrees, arcminute, arcsecond to a float degrees value.
-    """
-
+     """
```

Comment:
I don't think this should be indented.
```suggestion
    """
```